website/docs/getting-started/for-designers/partials/best-practices.md

```diff
@@ -1,87 +1,101 @@
 
 - It forces engineers to create custom classes and styles to override Helios styles. This can have unexpected long-term effects within the [CSS cascade](https://developer.mozilla.org/en-US/docs/Web/CSS/Cascade#:~:text=The%20cascade%20is%20an%20algorithm,a%20property%20on%20an%20element).
-- While being difficult to scale, overrides nudge the visual language of your product out of sync with Helios and with the rest of the HashiCorp product line.
+- In addition to being difficult to scale, overrides nudge the visual language of your product out of sync with Helios and with the rest of the HashiCorp product line.
 
 ### Resizing components
 
-Helios components are designed and built to be as layout-agnostic as possible, meaning the task of laying out components is left to the consumer.
+Helios components are designed and built to layout-agnostic, meaning the task of laying out components in a design is left to the consumer.
 
-Unless specifically mentioned in the documentation, the component in code will fill the parent container it is used in. For designers, it's helpful to use [auto layout](https://help.figma.com/hc/en-us/articles/5731482952599-Using-auto-layout#:~:text=You%20can%20add%20auto%20layout%20to%20a%20selected%20frame%2C%20component,and%20select%20Add%20Auto%20layout) wherever possible which closely mimics [CSS Flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/). Helios Figma components are flexible and can be set to `fill container` within an auto layout container, which will closely replicate the experience in a development environment.
+Unless specifically mentioned in the documentation, the component in code will fill the parent container it is used in. For designers, it's helpful to use [auto layout](https://help.figma.com/hc/en-us/articles/5731482952599-Using-auto-layout#:~:text=You%20can%20add%20auto%20layout%20to%20a%20selected%20frame%2C%20component,and%20select%20Add%20Auto%20layout) wherever possible which closely mimics [CSS Flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) and other layout mechanisms in the browser. Helios Figma components are flexible and can be set to `fill container` within an auto layout container, which replicates the experience of a development environment.
 
 !!! Warning
 
-**Fixed width components:** Setting a component to a fixed width in Figma is akin to setting an explicit `width` value on a component or element in code. This generally goes against fluid and responsive development best practices and should be avoided in Figma. Instead, use auto layout wherever possible.
+**Fixed width components**
+
+Setting a component to a fixed width in Figma is akin to setting an explicit `width` value on a component or element in code. This generally goes against fluid and responsive best practices and should be avoided in Figma. Instead, use auto layout wherever possible.
```

Comment:
```suggestion
Setting a component to a fixed width in Figma is akin to setting an explicit `width` value on a component or element in code. This generally goes against fluid and responsive best practices and should be avoided. Instead, use auto layout wherever possible.
```

Reply:
I'd actually argue here that the 'in Figma' portion here is useful, because this specific topic is aimed at designers. Though, the language probably overlaps a bit; fixed is a common term in fixed, probably a bit debatable on the engineering side.

Reply:
I vote for keeping "in Figma" as well since it provides the necessary context.

Reply:
To me the "in Figma" seemed repetitive and also it's a best practice for both Figma components and components in code so that's why I suggested removing.